.gitignore

```diff
@@ -41,6 +41,7 @@ tools/mips_to_c/
 *.jpg
 !*_custom*
 .extracted-assets.json
+ctx.c.m2c
```

Comment:
I added this because m2c started generating this file as a cache file or something like that.
I'm taking a look at the gitignore right now, and it seems like some other PR already added it, so I'll remove this

Reply:
Hmm, I hadn't noticed it doing so... seemed as slow as before.